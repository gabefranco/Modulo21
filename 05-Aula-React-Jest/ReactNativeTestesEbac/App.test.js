import { expect } from '@jest/globals';
import react from 'react';
import renderer from 'react-test-renderer';
import App from './App';

test('01 - Tela Ã© renderizada corretamente', () => {
    const tree = renderer.create(<App />).toJSON()
    expect(tree).toMatchSnapshot()
});