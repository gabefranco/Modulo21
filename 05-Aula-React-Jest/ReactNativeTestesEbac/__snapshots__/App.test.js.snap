// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`01 - Tela Ã© renderizada corretamente 1`] = `
<View
  style={
    {
      "alignItems": "center",
      "backgroundColor": "#fff",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text>
    " Meu primeiro APP React Native "
  </Text>
</View>
`;
